install.sh

```diff
@@ -356,11 +356,9 @@ parse_flags ()
             --with-qt)
                 with_qt='1'
                 ;;
-            -?*)
-                echo "warning.  unknown flag : $1" 1>&2
-                ;;
             *)
-                break
+                echo -e "\nUsage: "${0}" [options]\n\nOptions:\n\n--develop       code remains editable in place\n--python, -p    python version: python2 or python3 (default: python2)\n--with-qt       build the Qt GUI (incompatible with python2)\n"
```

Comment:
--python/-p allows not only python2/python3, but also more specific versions like python2.7 or python3.6 (was necessary for my environment).

Reply:
Thanks for pointing that out. I've removed the options for `-p` and just left the default as an example. If someone has a better suggestion just let me know.

I'll squash the commits once I'm sure that there's no more to be made.